lib/sarif_v_2_1_0.atd

```diff
@@ -182,6 +182,7 @@ type attachment <ocaml attr="deriving show,eq,ord"> = {
 
 type artifact_change <ocaml attr="deriving show,eq,ord"> = {
   artifact_location
```

Comment:
might make sense to use directly artifactLocation for the field name, so you don't have to 
add this <json name="..."> annotation.
It removes distance between the type definition and the actual JSON output which I think is good.

Reply:
> might make sense to use directly artifactLocation for the field name, so you don't have to add this annotation. It removes distance between the type definition and the actual JSON output which I think is good.

Yes this could work although there is no clear naming convention on atdgen types, I decided to stick to snake case so it looks more like OCaml types

Reply:
true, but in my experience, better to match as close as possible the original spec and remove distance. Also better to keep those atdgen spec small, so all those <json ...> and other annotations are cluttering the spec IMHO.

Reply:
Thanks for the feedback, I will think about it and look at some potential refactoring